internal/deployer/random.go

```diff
@@ -0,0 +1,14 @@
+// Package deployer for project deployment
+package deployer
+
+import "math/rand"
+
+const letterBytes = "abcdefghijklmnopqrstuvwxyz"
+
+func randString(n int) string {
```

Comment:
What is this function used for? is it for a gateway domain prefix? if so make it more explicit please